### PR TITLE
Fixes Chatbot Image Sizing

### PR DESCRIPTION
**Pull request by dawoodkhan82**
## Description
Increases size of uploaded images for chatbot.

Closes: #7769

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

**Comment by abidlabs** [comment]
I wonder if this should be configurable?

File: js/chatbot/shared/ChatBot.svelte [hunk 1]
```diff
@@ -325,7 +325,10 @@
 
 	.message-wrap > div :not(.avatar-container) :global(img) {
 		border-radius: 13px;
+		margin: var(--size-2);
+		width: 400px;
 		max-width: 30vw;
+		max-height: auto;
```
Comment (pngwn) [inline js/chatbot/shared/ChatBot.svelte:331]: Do you think we should have different sized avatars for mobile / desktop?

Comment (dawoodkhan82) [reply js/chatbot/shared/ChatBot.svelte:331]: This actually doesn't affect avatar images, only images uploaded into the chatbot.

Are the avatar images on mobile too large right now?

Comment (pngwn) [reply js/chatbot/shared/ChatBot.svelte:331]: Ah sorry. My mistake. 

No I think avatar images are fine on mobile.

**Comment by dawoodkhan82** [comment]
> I wonder if this should be configurable?

@abidlabs I don't think so, do we offer that much control over sizing on other components? I think once we add the option to click and zoom into any image in the chatbot, that would solve any "sizing" issues.

**Review by pngwn — APPROVED** [review]
Looks great to me! Mentioned in person that it might be worth checking the behaviour on very small screens but otherwise good to go. Thanks @dawoodkhan82!

**Comment by dawoodkhan82** [comment]
Thanks for the review @pngwn. Seems like it reacts fine to small screen sizes

Screenshot 2024-04-15 at 4 48 08 PM

**Comment by freddyaboulton** [comment]
Nice test image @dawoodkhan82 hehe

**Comment by yvrjsharma** [comment]
Hello @dawoodkhan82, thanks for looking into this. I have noticed that the image resizing works perfectly for large images and videos. However, I have observed that for smaller images, they appear stretched. Is there any way to prevent this from happening?

https://github.com/gradio-app/gradio/assets/48665385/8cb65bd3-7ef2-4f90-a8d6-1f4c81d7ca9a

**Pull request merged**